src/com/rcv/ResultsWriter.java

```diff
@@ -479,6 +476,20 @@ private int addHeaderRows(
     DateFormat dateFormat = new SimpleDateFormat("M/d/yyyy");
     // string for formatted date
     String dateString = dateFormat.format(new Date());
+    // string indicating single- or multi-winner
+    String electionType = config.numberOfWinners() > 1 ? "multi-winner" : "single-winner";
+    // number reflecting threshold percentage to win
+    BigDecimal thresholdPercentage = config.divide(
+        new BigDecimal(100),
+        new BigDecimal(config.numberOfWinners() + 1)
+    );
+    // string for threshold percentage
+    String thresholdString = thresholdPercentage.toString() + "%";
+    // formula string
+    String formula = config.numberOfWinners() > 1 ?
+        "More than 1/" + (config.numberOfWinners() + 1) + " of total votes cast for office" :
+        "Half of total votes cast for office + 1";
```

Comment:
Make wording consistent between these formulas, e.g. either "more than half of total votes cast for office" or "More than 1/* of total votes cast for office + 1".

Reply:
OK, fair enough. I was trying to preserve the language that Whitney had given us, but yeah, we should just be consistent.